rider.rb

```diff
@@ -0,0 +1,9 @@
+class Rider
+  def initialize(name)
+    @name = name
+  end
+
+  def make_magic
+    @name = "Yer a wizard #{name}"
```

Comment:
When we make a rider magic, should we maintain their non-magic name for use in the non-magic world?